test/test_dynamics.jl

```diff
@@ -261,7 +261,7 @@ function stratified_fluid_remains_at_rest_with_tilted_gravity_buoyancy_tracer(ar
     grid = RectilinearGrid(arch, FT, topology=topo, size=(1, N, N), extent=(L, L, L))
 
     g̃ = (0, sind(θ), cosd(θ))
-    buoyancy = Buoyancy(model=BuoyancyTracer(), gravity_unit_vector=g̃)
+    buoyancy = Buoyancy(model=BuoyancyTracer(), gravity_unit_vector=.-g̃)
```

Comment:
```suggestion
    buoyancy = Buoyancy(model=BuoyancyTracer(), gravity_unit_vector=-g̃)
```

Reply:
This doesn't work since it's a tuple:

```julia
julia> θ = 45 # degrees
45

julia> g̃ = (0, sind(θ), cosd(θ))
(0, 0.7071067811865476, 0.7071067811865476)

julia> -g̃
ERROR: MethodError: no method matching -(::Tuple{Int64, Float64, Float64})
Closest candidates are:
  -(::Tuple, ::Oceananigans.AbstractOperations.AbstractGridMetric, ::Oceananigans.Fields.AbstractField) at ~/repos/Oceananigans.jl/src/AbstractOperations/binary_operations.jl:119
  -(::Tuple, ::Function, ::Oceananigans.Fields.AbstractField) at ~/repos/Oceananigans.jl/src/AbstractOperations/binary_operations.jl:116
  -(::Tuple, ::Number, ::Number) at ~/repos/Oceananigans.jl/src/AbstractOperations/binary_operations.jl:113
  ...
Stacktrace:
 [1] top-level scope
   @ REPL[19]:1
 [2] top-level scope
   @ ~/.julia/packages/CUDA/BbliS/src/initialization.jl:52
```

I think we prefer using Tuples since they work better on GPUs, no? If not, I can change `g` to be an Array and apply your suggestion.